readme.md

```diff
@@ -158,7 +164,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:**  Whichever value is set to required and is omitted in the request will throw an error. You can use a try/catch function to output a custom error message
```

Comment:
You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.